library/styleguideComponents/UniversalSearch/PostItemSearchResult.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LabelText from './labelText';
+import DescriptionText from './descriptionText';
+import {
+    ResultItemContainer,
+    IconContainer,
+    OptionText,
+    PostContainer,
+    StyledImage,
+    PostText,
+    Information,
+} from './styled';
+
+const ItemSearchResult = props => {
+    const { icon, title, description, image, date, author } = props;
+    const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
```

Comment:
remember, fixed constants that do not depend on function params can be outside, thus avoiding recreating a new object each time